WordPress/Classes/ViewRelated/Reader/Select Interests/ReaderSelectInterestsViewController.swift

```diff
@@ -383,6 +385,7 @@ extension ReaderSelectInterestsViewController: UICollectionViewDelegateFlowLayou
 
 // MARK: - ReaderInterestsDataDelegate
 extension ReaderSelectInterestsViewController: ReaderInterestsDataDelegate {
+    @MainActor
```

Comment:
Kind of the same comment here, I would think that calling `reloadData` on a background thread would've caused a warning/crash. So my guess is this was already being called on the main thread.

Reply:
(Addressed in https://github.com/wordpress-mobile/WordPress-iOS/pull/23158#discussion_r1593783869)